moved/src/types/transactions.rs

```diff
@@ -456,7 +476,6 @@ mod tests {
         rlp_roundtrip(&Bytes::from_static(&hex!("02f86f82a45580808346a8928252089465d08a056c17ae13370565b04cf77d2afa1cb9fa8806f05b59d3b2000080c080a0dd50efde9a4d2f01f5248e1a983165c8cfa5f193b07b4b094f4078ad4717c1e4a017db1be1e8751b09e033bcffca982d0fe4919ff6b8594654e06647dee9292750")));
     }
 
-    #[cfg(test)]
```

Comment:
Should this be kept?

Reply:
It was redundant because it appears in a block that is already marked as `#[cfg(test)]`.